quora-api/src/main/java/com/upgrad/quora/api/exception/RestExceptionHandler.java

```diff
@@ -0,0 +1,36 @@
+package com.upgrad.quora.api.exception;
```

Comment:
This file is **not required**, as we already have all the exception defined in 
**path** : quora-service/src/main/java/com/upgrad/quora/service/exception
and it can be **imported in quora-api**, 
and other exceptions have also to be **created in this path only**